wp-content/themes/missilethreat/assets/_scss/layout/_footer.scss

```diff
@@ -1 +1,117 @@
 @use 'abstracts' as *;
+
+.footer {
+  @extend %font-body-short-md;
+  @include full-width-background($color-primary-blue-100);
+  padding-top: rem(40px);
+  padding-bottom: rem(16px);
+  color: $color-text-light-170;
+  background-color: $color-primary-blue-100;
+
+  &__contact {
+    display: flex;
+    flex-wrap: nowrap;
+    margin-top: rem(32px);
+    margin-bottom: rem(32px);
+  }
+
+  &__address,
+  &__news-letter,
+  &__nav-side,
+  &__copyright {
+    @extend %font-body-short-sm
+  }
```

Comment:
This font group is already applied to the whole footer - however, each of these have different font groups.
Address - `%font-ui-text-md`
Newsletter and navigation - `%font-body-short-md`
Copyright - `%font-ui-text-md`